docs/doc.main

```diff
@@ -120,7 +121,7 @@
      "InvariantTheory/reductive_groups.md",
   ],
 
-  "Geometry" => [
+  "Algebraic Geometry" => [
```

Comment:
I've renamed this as requested in another conversation by last week by @micjoswig (and @wdecker agreed).